### PR TITLE
added summary tile(view team advance request revamp)

### PR DESCRIPTION
**Pull request by sandy-fyle**
added a summary tile in view team advance request page

image
image

### CODE REVIEW

**Comment by Dimple16** [comment]
the second screenshot looks scary, we need to add ellipsis in such cases,
let the text go up to 2 lines, then add ellipsis

**Comment by Dimple16** [comment]
I guess the edit icon of approvers is missing

File: src/app/fyle/view-team-advance/view-team-advance.page.scss [hunk 1]
```diff
@@ -11,7 +11,6 @@
 
   &--card {
     padding: 0 16px;
-    margin: 16px;
     border: 1px solid #e5e5e5;
```
Comment (Dimple16) [inline src/app/fyle/view-team-advance/view-team-advance.page.scss:14]: use color variables

**Review by Dimple16 — CHANGES_REQUESTED** [review]

**Comment by sandy-fyle** [comment]
> 

okay


> I guess the edit icon of approvers is missing

I have created separate task for it, will create PR soon

File: src/app/fyle/view-team-advance/view-team-advance.page.scss (hunk 1 above)
Comment (sandy-fyle) [reply src/app/fyle/view-team-advance/view-team-advance.page.scss:14]: okay

File: src/app/shared/components/summary-tile/summary-tile.component.scss [hunk 2]
```diff
@@ -27,6 +27,9 @@ $advance_paid_color: darken(#5cb85c, 15%);
       color: $black;
       font-size: 18px;
       font-weight: 500;
+      white-space: nowrap;
+      overflow: hidden;
+      text-overflow: ellipsis;
```
Comment (Dimple16) [inline src/app/shared/components/summary-tile/summary-tile.component.scss:32]: use ellipsis pipe

**Review by Dimple16 — CHANGES_REQUESTED** [review]

File: src/app/shared/components/summary-tile/summary-tile.component.html [hunk 3]
```diff
@@ -2,7 +2,9 @@
   <div class="d-flex">
     <div class="summary-tile--card-container__block">
       <div *ngIf="category" class="summary-tile--card-container__category">{{ category }}</div>
-      <div *ngIf="purpose" class="summary-tile--card-container__purpose">{{ purpose }}</div>
+      <div *ngIf="purpose" class="summary-tile--card-container__purpose">
+        {{ purpose.length > 25 ? (purpose | slice: 0:25) + '..' : purpose }}
```
[the remark this answers is not in the document]
Comment (Dimple16) [reply src/app/shared/components/summary-tile/summary-tile.component.html:6]: you can directly use `purpose | ellipsis: 25` 
why do we need a slice here? please check the existing ellipsis pipe usage and make similar changes

[the remark this answers is not in the document]
Comment (sandy-fyle) [reply src/app/shared/components/summary-tile/summary-tile.component.html]: > 

image

**Comment by sandy-fyle** [comment]
> added a summary tile in view team advance request page
> 
> image image

image

**Review by Dimple16 — APPROVED** [review]

**Pull request closed without merging**